libraries/ESP8266WebServer/examples/FSBrowser/extras/index_htm.h

```diff
@@ -0,0 +1,525 @@
+unsigned char index_htm_gz[] = {
```

Comment:
There should be a BigPhatComment somewhere explaining that modifications to index.htm should be accompanied by an update to this.
In fact, that comment should be in in the .ino, inside index.htm, and in here, at the minimum.